lib/Heartbeat/LED.cpp

```diff
@@ -58,6 +58,8 @@ void LED::setState(boolean state) {
 
 boolean LED::getState() { return digitalRead(_pin) ^ !_activeHigh; }
 
+LED::~LED() {}
```

Comment:
Isn't it noise, that will be changed later again.

Reply:
See https://github.com/puuu/MQTT433gateway/pull/15#discussion_r154341168